app/controllers/assessment/autograde.rb

```diff
@@ -119,15 +120,16 @@ def regradeAll
           redirect_to([@course, @assessment, :submissions]) && return
         elsif job < 0 # autograding failed
           failed_jobs += 1
-          failed_list += "#{@submission.filename}: autograding error.<br>"
+          failed_list += "#{submission.filename}: #{flash[:error]}.<br>"
```

Comment:
What does this behavior look like? Is it necessary to have a flash inside of a message that will be itself flashed on line 132?